tests/test_scalar.py

```diff
@@ -0,0 +1,102 @@
+import pytest
+
+from moztelemetry.scalar import Scalar
+
+
+def test_int_scalar():
+    name, value = 'telemetry.test.unsigned_int_kind', 8
+    scalar = Scalar(name, value)
+
+    assert scalar.get_value() == value
+    assert scalar.get_name() == name
+    assert scalar.get_definition().get('kind') == 'uint'
+    assert Scalar.REQUIRED_FIELDS.issubset(scalar.get_definition().viewkeys())
+
+
+def test_string_scalar():
+    name, value = 'telemetry.test.string_kind', 'string'
+    scalar = Scalar(name, value)
+
+    assert scalar.get_value() == value
+    assert scalar.get_name() == name
+    assert scalar.get_definition().get('kind') == 'string'
+    assert Scalar.REQUIRED_FIELDS.issubset(scalar.get_definition().viewkeys())
+
+
+def test_boolean_scalar():
+    name, value = 'telemetry.test.boolean_kind', True
+    scalar = Scalar(name, value)
+
+    assert scalar.get_value() == value
+    assert scalar.get_name() == name
+    assert scalar.get_definition().get('kind') == 'boolean'
+    assert Scalar.REQUIRED_FIELDS.issubset(scalar.get_definition().viewkeys())
+
+
+def test_int_addition():
+    name, value = 'telemetry.test.unsigned_int_kind', 8
+    scalar = Scalar(name, value) + Scalar(name, value)
+
+    assert scalar.get_value() == value * 2
+    assert scalar.get_name() == name
+    assert Scalar.REQUIRED_FIELDS.issubset(scalar.get_definition().viewkeys())
+
+
+def test_string_addition():
+    name, value = 'telemetry.test.string_kind', 'string'
+
+    with pytest.raises(AttributeError):
+        Scalar(name, value) + Scalar(name, value)
+
+
+def test_boolean_addition():
+    name, value = 'telemetry.test.boolean_kind', True
+
+    with pytest.raises(AttributeError):
+        Scalar(name, value) + Scalar(name, value)
+
+
+def test_release_channel():
+    name, value = 'telemetry.test.unsigned_int_kind', 8
+    scalar = Scalar(name, value, channel='release')
+
+    assert scalar.get_value() == value
+    assert scalar.get_name() == name
+    assert Scalar.REQUIRED_FIELDS.issubset(scalar.get_definition().viewkeys())
+
+
+def test_release_revision():
+    name, value = 'telemetry.test.unsigned_int_kind', 8
+    scalar = Scalar(name, value,
+        revision='https://hg.mozilla.org/releases/mozilla-release/rev/tip')
+
+    assert scalar.get_value() == value
+    assert scalar.get_name() == name
+    assert Scalar.REQUIRED_FIELDS.issubset(scalar.get_definition().viewkeys())
+
+
+def test_release_url():
+    name, value = 'telemetry.test.unsigned_int_kind', 8
+    scalar = Scalar(name, value,
+        scalars_url='https://hg.mozilla.org/releases/mozilla-release/raw-file/tip/toolkit/components/telemetry/Scalars.yaml')
```

Comment:
It would be great if you could add a copy of Scalars.yml to tests/data and use that for your tests